examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -107,7 +176,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Solve using Remote QPU backend "
+    "## Solving using Remote QPU backend "
```

Comment:
We'll need a little narration here, too.

Reply:
Done.